library/libwaku.h

```diff
@@ -92,6 +92,22 @@ int waku_relay_unsubscribe(void* ctx,
                            WakuCallBack callback,
                            void* userData);
 
+int waku_filter_subscribe(void* ctx,
+                          const char* pubSubTopic,
+                          const char* contentTopics,
+                          WakuCallBack callback,
+                          void* userData);
+
+int waku_filter_unsubscribe(void* ctx,
+                            const char* pubSubTopic,
+                            const char* contentTopics,
+                            WakuCallBack callback,
+                            void* userData);
+
+int waku_filter_unsubscribe_all(void* ctx,
+                                WakuCallBack callback,
+                                void* userData);
+
```

Comment:
I think we need ping as well! Is it left out intentionally?
Without ping the client is forced to use subscribe all the time (in each 1-2 minutes) which is significantly higher load on service node than just check with ping.

Reply:
Thanks for that and very good point!
IMO, I think it is better to perform the ping internally and avoid the `libwaku` user handling that. I will submit another commit soon

Reply:
Do you mean, hide ping behind the scene? Let say ping every minute if any subscription is added and between unsubscribe_all?

Reply:
I agree!